pkg/sectigo/profiles/profiles.go

```diff
@@ -1,4 +1,4 @@
-package sectigo
+package profiles
```

Comment:
This is a temporary workaround so that `sectigo.go` and `config.go` don't reference each other.

Reply:
This seems like a bit much for just a few constants. I'd like to refactor as follows:

1. Move `config.SectigoConfig` to `sectigo.Config` in `pkg/sectigo/config.go` (including validation)
2. Replace the profiles.go
3. Make the `config.Sectigo` property a `sectigo.Config` 

I think this should solve the problem of the recursive import and I think that `sectigo.Config` is much neater and decoupling anyway.